lbry/lbry/wallet/database.py

```diff
@@ -52,26 +52,37 @@ def txo_to_row(self, tx, address, txo):
             row['claim_name'] = txo.claim_name
         return row
 
-    async def get_txos(self, **constraints) -> List[Output]:
-        my_account = constraints.get('my_account', constraints.get('account', None))
-
-        txos = await super().get_txos(**constraints)
+    async def get_txos(self, my_accounts, filtering_accounts=None, **constraints) -> List[Output]:
+        constraints.pop('account', None)
```

Comment:
Why is this just getting removed? Can it just not be passed in altogether if it's just going to be pop'ed off?

Reply:
This was there as fallback, in case some function is passing `account` which was not caught during the programming, I probably put in for debug. But can remove.